app/assets/javascripts/omni-nav-v2.js

```diff
@@ -105,9 +106,21 @@ var OmniNav2 = (function() {
 
     var syncLinkWidths = function() {
       var width = $('#js-off-canvas-nav > ul').width();
-      offCanvasLinks.css('width', width);
+      $offCanvasLinks.css('width', width);
     };
 
+    var enableMenusToggle = function() {
+      // Enables toggle to slide main/umbrella menus back and forth.
+      $('a.toggle-menu-label').on('click', function(e) {
+        console.debug('toggle menus now!');
```

Comment:
Do we have a standard for committing debug outputs?

Reply:
Yes, I'll remove that.